src/main/java/frc/robot/subsystems/shooter/commands/bits/CheckVision.java

```diff
@@ -0,0 +1,48 @@
+package frc.robot.subsystems.shooter.commands.bits;
+
+import edu.wpi.first.math.filter.LinearFilter;
+import edu.wpi.first.wpilibj2.command.ParallelRaceGroup;
+import edu.wpi.first.wpilibj2.command.RunCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.subsystems.hood.Hood;
+import frc.robot.subsystems.shooter.Shooter;
+import frc.robot.subsystems.shooter.commands.Shoot;
+import frc.robot.utils.PhotonVisionModule;
+import org.photonvision.PhotonCamera;
+import org.photonvision.PhotonUtils;
+
+import static frc.robot.Constants.Vision.*;
+
+public class CheckVision extends SequentialCommandGroup {
+
+
+    public CheckVision(Shooter shooter, Hood hood, PhotonCamera photonCamera, LinearFilter linearFilter) {
+        addCommands(
+                new ParallelRaceGroup(
```

Comment:
Just in case, turn on the leds of the vision and the wait for 0.5 seconds (let the vision "warm up")